develop-docs/api-server/application-domains/feature-flags/flagpole.mdx

```diff
@@ -240,3 +240,9 @@ you remove the feature option.
 
 To unset your feature, comment out or remove your feature config from the
 `option` object, and rerun the `getsentry configoptions` command above.
+
+## Removing Flagpole features
+Because our options automator CI relies on the feature definitions in both Sentry and GetSentry, flags must be removed in a specific order to avoid causing test failures:
```

Comment:
```suggestion
## Removing Flagpole features

Because our options automator CI relies on the feature definitions in both Sentry and GetSentry, flags must be removed in a specific order to avoid causing test failures:

```